### ISSUE TITLE
Data Import - Code snippet link missing text

### ISSUE THREAD

**Issue by yan-ilumino** [mine-cetinkaya-rundel/accessible-cheatsheets#4]
**Page**: https://mine-cetinkaya-rundel.github.io/accessible-cheatsheets/data-import/data-import-accessible.html

**Issue**

All of the code snippet anchor elements are missing text.

`<a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>`

**How to fix it**

If you have access to change the code, remove the empty link or provide text within the links that describes the functionality and/or target of that link.

**Note**:

It's most likely a false positive as it won’t be announced to the screen reader user and won’t be accessible to the user that navigates your page with a keyboard only, but an automated scanning tool will call them out and can cause frivolous lawsuits.

**Success Criteria**

- [2.4.4 Link Purpose (In Context) (Level A)](https://www.w3.org/WAI/WCAG21/quickref/#link-purpose-in-context)
